packages/runtime-dom/src/jsx.ts

```diff
@@ -46,7 +46,7 @@ type Booleanish = boolean | 'true' | 'false'
 type Numberish = number | string
 
 // All the WAI-ARIA 1.1 attributes from https://www.w3.org/TR/wai-aria-1.1/
-interface AriaAttributes {
+export interface AriaAttributes {
```

Comment:
Why export it?

Reply:
because i need types i will use in Oku. (https://github.com/oku-ui/primitives) Also, this is exported on the react.

Reply:
@productdevbook 

![image](https://github.com/vuejs/core/assets/13535256/90c0c415-9ee5-48c4-8c9b-eb21663a2d50)

Reply:
https://github.com/radix-ui/primitives/blob/eca6babd188df465f64f23f3584738b85dba610e/packages/react/roving-focus/src/RovingFocusGroup.tsx#L36

The last place you showed. Base developers always need this data.